packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "version": "0.1.0",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "$(npm bin)/nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
```

Comment:
Just using the global name (i.e. `nyc`) works in npm scripts. This is a quick fix to have commands work on Windows.